be/src/runtime/load_channel_mgr.h

```diff
@@ -58,6 +59,14 @@ class LoadChannelMgr {
     // cancel all tablet stream for 'load_id' load
     Status cancel(const PTabletWriterCancelRequest& request);
 
+    void refresh_mem_tracker() {
```

Comment:
why do we need refresh memtracker periodly?

Reply:
Because the normal mem tracker used by LoadChannelMgr has no parent-child relationship, the consumption of the MemTable tracker will not be automatically synchronized to the LoadChannelMgr tracker.